rubin_scheduler/scheduler/surveys/surveys.py

```diff
@@ -196,6 +193,9 @@ def __init__(
         search_radius=None,
         alt_max=-9999,
         az_range=-9999,
+        target_name="",
```

Comment:
Should target_name be None or should it be ""? "" is almost None, but has different behavior if we're checking for it downstream.